R/Table.R

```diff
@@ -50,6 +50,15 @@ odbc_write_table <-
       dbExecute(conn, sql)
     }
 
+    fieldDetails <-
+      tryCatch({
+        details <- odbcConnectionColumns(conn, name)
+        datails <- details[match(names(values), details$column_name)]
```

Comment:
This could just be

```r
datails <- details[names(values)]
```

But do we need to reorder the columns at all?

Reply:
Jim, the intention here was to:
* Subset the rows of the output of SQLColumns to those (columns) we are writing to / we need to describe.
* I thought some care was needed with the ordering - AFAICT there is no guarantee that the columns in `values` (and therefore the parameters in the INSERT statement) are ordered the same way as the columns in the table we are writing to (i.e. the rows in `details`).

Having said that, I think part of the confusion is the use of `ordinal_position` to index the parameters we are describing - this seems incorrect.  After re-ordering the rows in `details` to match the column order in `values` as is done here, the index that gets passed `nanodbc::statement::describe_parameters` should be `1:ncol(values)`